src/tests/ftest/pool/attribute.py

```diff
@@ -101,7 +103,7 @@ def setUp(self):
                 self.hostlist = self.params.get("test_machines", '/run/hosts/*')
                 self.hostfile = write_host_file.write_host_file(self.hostlist,
                                                                 self.workdir)
-
+                self.agent_sessions = run_agent(basepath, self.hostlist)
```

Comment:
Just a suggestion (so ignore if you wish), but instead of just adding the agent here, you could just port this test to [apricot.TestWithServers](https://github.com/daos-stack/daos/pull/402).

An example of a test ported to [apricot.TestWithServers](https://github.com/daos-stack/daos/pull/402) can be found [here](https://github.com/daos-stack/daos/pull/444/files#diff-9cf69dec1813bc71678c4aa6a3729a87).  So as you can see, it's pretty straightforward.